tests/azure/templates/playbook_tests.yml

```diff
@@ -30,11 +31,11 @@ jobs:
   - script: |
       pip install \
         "molecule[docker]>=3" \
-        "ansible${{ parameters.ansible_version }}" \
-        jmespath \
-        pytest \
-        pytest-split-tests
-    displayName: Install dependencies
+        "ansible${{ parameters.ansible_version }}"
+    displayName: Install molecule and Ansible
+
+  - script: pip install -r tests/requirements.txt
```

Comment:
I prefer a centralized configuration. When it is not possible, I don't like spreading configuration files through multiple directories, unless required to.

I see several projects using `requirements-dev.txt` (and `tests` is a part of `dev`), or we could also use `test_requires` from setuptools (either on `setup.py` or `setup.cfg`).

As we discussed earlier, I don't have a hard opinion about it. But having configuration files spread across the source tree it the way I like it less.

Reply:
I can change it to `/requirements-tests.txt` and `requirements-dev.txt` if you like. I'd used inside tests directory to use the name space. I don't see much of a difference in either case. Let me know.

Reply:
I think we might use a third opinion here... @t-woerner, @varunmylaraiah do you have a opinion or preference on this?

Reply:
I think requirements-tests.txt and requirements-dev.txt are good.